pkg/power/uptime.go

```diff
@@ -57,7 +57,7 @@ func (u *Uptime) SendNow() (types.Hash, error) {
 		return types.Hash{}, errors.Wrap(err, "failed to get uptime")
 	}
 
-	return sub.UpdateNodeUptime(u.id, uptime)
+	return sub.UpdateNodeUptimeV2(u.id, uptime, uint64(time.Now().Unix()))
```

Comment:
can you please add in the documentation the change in behavior

Reply:
you are right, done